### ISSUE TITLE
Error: Unable to resolve module: <path>/empty-module.js

### ISSUE THREAD

**Issue by jamonholmgren** [microsoft/rnx-kit#1605]
### What happened?

When updating Ignite to the latest React Native, enabling symlinks via `MetroSymlinksResolver` results in an error:

```
error: Error: Unable to resolve module /Users/jh/Code/ignite-cli/boilerplate/node_modules/metro-runtime/src/modules/empty-module.js from /Users/jh/Code/ignite-cli/boilerplate/_: /Users/jh/Code/ignite-cli/boilerplate/node_modules/metro-runtime/src/modules/empty-module.js could not be found
```

We're using `MetroSymlinksResolver` to support monorepos and pnpm. Disabling it is not ideal.

<img width="200" alt="error shown in ios simulator" src="https://user-images.githubusercontent.com/1479215/172224426-503d49bd-029a-4a67-b9a8-1c95344a68b6.png">

### Affected Package

@rnx-kit/metro-resolver-symlinks

### Version

0.1.20

### Which platforms are you seeing this issue on?

- [ ] Android
- [X] iOS
- [ ] macOS
- [ ] Windows

### System Information

```true
System:
    OS: macOS 12.4
    CPU: (10) arm64 Apple M1 Max
    Memory: 2.34 GB / 64.00 GB
    Shell: 5.8.1 - /bin/zsh
  Binaries:
    Node: 14.19.1 - /opt/homebrew/opt/node@14/bin/node
    Yarn: 1.22.19 - /opt/homebrew/bin/yarn
    npm: 6.14.16 - /opt/homebrew/opt/node@14/bin/npm
    Watchman: 2022.03.21.00 - /opt/homebrew/bin/watchman
  Managers:
    CocoaPods: 1.11.3 - /opt/homebrew/bin/pod
  SDKs:
    iOS SDK:
      Platforms: DriverKit 21.4, iOS 15.5, macOS 12.3, tvOS 15.4, watchOS 8.5
    Android SDK: Not Found
  IDEs:
    Android Studio: 2021.2 AI-212.5712.43.2112.8512546
    Xcode: 13.4.1/13F100 - /usr/bin/xcodebuild
  Languages:
    Java: 11.0.14 - /Users/jh/.sdkman/candidates/java/current/bin/javac
  npmPackages:
    @react-native-community/cli: Not Found
    react: 17.0.2 => 17.0.2
    react-native: 0.68.2 => 0.68.2
    react-native-macos: Not Found
  npmGlobalPackages:
    *react-native*: Not Found
```
```


### Steps to Reproduce


To reproduce (ignore the misleading branch name, it's correct):

```
git clone https://github.com/infinitered/ignite.git
cd ./ignite/boilerplate
git checkout issue-1942-react-state-update-bug
yarn
yarn ios
```


### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by jamonholmgren** [comment]
#1567 could be related.

**Comment by tido64** [comment]
I followed the instructions above and can't reproduce this.

![image](https://user-images.githubusercontent.com/4123478/172316629-f12f2675-606f-4f9a-97fb-74aa7f5888ab.png)

Can you try again and verify that the steps are correct?

**Comment by tido64** [comment]
The fix landed in [0.1.21](https://github.com/microsoft/rnx-kit/releases/tag/%40rnx-kit%2Fmetro-resolver-symlinks%400.1.21).

**Comment by jamonholmgren** [comment]
Thanks @tido64!
